core/cmd/wandb-core/main.go

```diff
@@ -23,65 +22,60 @@ func init() {
 }
 
 func main() {
-	portFilename := flag.String(
-		"port-filename",
-		"port_file.txt",
-		"filename for port to communicate with client",
-	)
-	pid := flag.Int("pid", 0, "pid")
-	debug := flag.Bool("debug", false, "debug mode")
-	noAnalytics := flag.Bool("no-observability", false, "turn off observability")
-	// todo: remove these flags, they are here for backward compatibility
-	serveSock := flag.Bool("serve-sock", false, "use sockets")
+	// Flags to control the server
+	portFilename := flag.String("port-filename", "port_file.txt", "filename for port to communicate with client")
+	pid := flag.Int("pid", 0, "pid of the process to communicate with")
+	debugLevel := flag.Bool("debug", false, "enable debug logging")
```

Comment:
naming nit:

```suggestion
	enableDebugLogging := flag.Bool("debug", false, "enable debug logging")
```

"debugLevel" sounds like it would be an integer or enum of some kind ("what's the debug level?")

Reply:
yeah, cause i wanted to convert it to debug level but the PR started  being too big, will return it to the debug which is just a bool